### ISSUE TITLE
NPE in uncaugt exception handling

### ISSUE THREAD

**Issue by zbynek** [gwtproject/gwt-core#17]
https://github.com/gwtproject/gwt-core/blob/master/gwt-core/src/main/java/org/gwtproject/core/client/GWT.java#L142
calls `original`, though it is (most likely) `null`.

Using `window.addEventListener("error")` instead of `window.onerror` would avoid this error.

**Comment by niloc132** [comment]
Right, I was suggesting this would be the simplest fix. I don't see an obvious reason to do one vs the other.
